Makefile

```diff
@@ -51,6 +51,11 @@ image_builder=
 build:
 	./hack/go-build.sh --image_registry ${image_registry} --image_name ${image_name} --image_tag ${image_tag} --image_builder ${image_builder}
 
+.PHONY: install
+image=
+install:
```

Comment:
please add extra info here, install is too generic, e.g. deploy-operator-on-ocp.

Reply:
Done and script renamed.